docs/UserGuide.md

```diff
@@ -360,371 +361,371 @@ edit /c 2 /n cs2040s
+- After editing the student, the `UI` will update to show you the student list of the class that the edited student is in.
 
 </box>
 
 <br>
 
-Return to [Table of Contents](#table-of-contents)
-
-## Student commands :mortar_board:
-
-### Adding a student : `add /s`
+### Marking a student present : `mark /s`
 
-Adds a new student to your existing class.
+Marks a student present for your current lesson.
 
-**Command Format:** `add /s STUDENT_NAME /c CLASS_INDEX`
+**Command Format:** `mark /s STUDENT_INDEX /c CLASS_NAME`
 
 **Examples:**
-* `add /s John /c 1`: Adds a new student named "John" into the first class on the displayed class list.
-* `add /s Bob /c 2`: Adds a new student named "Bob" into the second class on the displayed class list.
+* `mark /s 2 /c CS2103T` : Marks the second student in your class "CS2103T" as present.
+* `mark /s 3 /c CS2102` : Marks the third student in your class "CS2102" as present.
 
 **Sample Usage:**
 
-1. Assuming you want to add a student named "John" into the class "CS2103T", and the class index in the displayed list is "1".
+1. Assuming you want to mark the second student named "Jamie" in your class "CS2103T", whereby the class is positioned at index "1" in your class list, as present.
 2. Enter the following commands:
-   ````
-   list
-   add /s John /c 1
-   ````
+```
+view /c 1
+mark /s 2 /c CS2103T
+```
 3. The result box will display the following message:
-   ````
-   Added New Student: John; Id: A0000000Z; Memo:  to the class: CS2103T
-   ````
-4. You have successfully added the John into CS2103T. :tada:
+```
+Jamie successfully marked present!
+```
+
+4. You have successfully marked Jamie as present in CS2103T. :tada:
 
 <box type="info">
 
 **Information:**
-* Every student must be assigned to a class, i.e. a student should always appear in one of the classes.
-* An identical student (ie. all student information are the same) cannot be in more than 1 class at the same time.
-* A default id "A0000000Z" will be assigned to the student.
+
+* `CLASS_NAME` must be the name of a class that already exists.
 
 </box>
 
 <box type="warning">
 
 **Caution:**
-* The class to add the student into must already exist.
-* After adding the student, the `UI` will update to show you the student list of the class you added the new student into.
+
+* Your target student should not already be marked present.
 
 </box>
 
 <box type="tip">
 
 **Tip:**
-* If you want to see the student show up in the student list of the class, perform a [view command](#viewing-a-class-view-c) first. Be sure to remember the `CLASS_INDEX` though!
-* You can edit the student's id using the [edit student command](#editing-a-student-edit-s) once you have added the student!
-* You can also add a memo or class participation for a particular student using the same [edit student command](#editing-a-student-edit-s)!
+
+* If all your students are present, you can use our [markall](#marking-all-students-in-a-class-present-markall) command instead!
```

Comment:
for this hyperlink i think it should be #marking-all-students-in-a-class-present-markall-c

Reply:
Thanks for pointing out the mistake!